utils/ldap_client.py

```diff
@@ -0,0 +1,38 @@
+from ldap3 import Server, Connection, ALL
+from reconcile.config import get_config
+
+_client = None
+_base_dn = None
+
+
+def init(serverUrl):
+    global _client
+
+    if _client is None:
+        server = Server(serverUrl, get_info=ALL)
+        _client = Connection(server, None, None, auto_bind=True)
+
+    return _client
+
+
+def init_from_config():
+    global _base_dn
+
+    config = get_config()
+
+    serverUrl = config['ldap']['server']
+    _base_dn = config['ldap']['base_dn']
+
+    return init(serverUrl)
+
+
+def user_exists(username):
+
```

Comment:
perhaps remove this empty line?

Reply:
how about we follow the same approach for the ldap_client than for the gitlab_client? I think it's better so we can reuse  this logic for other ldap servers in the future.

This would mean:
- no `init` methods
- define class that applies contains the functionality
- new method that returns the instantiated class using the toml config `def get_ldap_client_from_config`

Reply:
after discussing with @maorfr , we believe this is not necessary, and we can leave it as is